README.md

```diff
@@ -10,32 +10,275 @@
+    * Student A1234567A has not completed task T1 of module CS2103 yet.
+    * But for some reason (e.g. human error) the task is marked as done.
+
+### Find a student: `find`
+
+Find the student in the module specified along with the student's information.
+
+Format: `find m/<module name> i/<student id>`
+
+Examples:
+* `find m/CS2103 i/A0123456A`:
+    * Finds and shows student with ID A0123456A from module CS2103
+
+### Go to homepage: `home`
+
+Return to the homepage which shows the complete list of modules and students along with their tasks.
+
+Format: `home`
+
+### Clear all information: `clear`
+
+Clear all information stored in the TAB application.
+
+Format: `clear`
+
+### Exit the application: `exit`
+
+Exit the TAB application.
+
+Format: `exit`
+
+## FAQ
+
+**Q**: How do I assign tasks to students?<br>
+**A**: Adding the student to the module will automatically add the tasks in the module to the student.<br>
+<br>
+**Q**: How do I see the full TAB after executing the `find` command?<br>
+**A**: Use the `home` command to return to the homepage showing the full TAB. <br>
+
+--------------------------------------------------------------------------------------------------------------------
+
+## Glossary
+
+**Alphanumeric**: Any characters from [0-9], [a-z], or [A-Z].<br>
+**TAB**: Teaching Assistant's Buddy.<br>
+**Tele handle**: A telegram user handle that must start with @, and followed by 5-32 characters containing only alphanumeric characters and the underscore '_'.
+
+
+--------------------------------------------------------------------------------------------------------------------
+## Command summary
+
+**Optional parameters** are surrounded by square brackets '[]' in Format, and may be omitted from Examples to show they are optional.<br>
+
+Action | Format, Examples
+--------|------------------
+**Add Module** | `add module m/<module name>` <br> e.g., `add module m/CS2103`
+**Add Student** | `add student m/<module name> i/<student id> n/<student name> t/<student tele handle> e/<student email>` <br> e.g., `add student m/CS2103 i/A0123456A n/John Doe t/@johndoe e/john.doe@u.nus.edu`
+**Add Task** | `add task m/<module name> ti/<task id> a/<task name> d/<task deadline>` <br> e.g., `add task m/CS2103 ti/T1 a/assignment1 d/2021-10-12 12:00`
+**Clear** | `clear` <br> e.g., `clear`
+**Delete Module** | `delete module m/<module name>` <br> e.g., `delete module m/CS2103`
+**Delete Student** | `delete student m/<module name> i/<student id>` <br> e.g., `delete student m/CS2103 i/A1234567A`
+**Delete Task** | `delete task m/<module name> ti/<task id>` <br> e.g., `delete task m/CS2103 ti/T1`
+**Edit Module** | `edit module m/<old module name> mn/<new module name>` <br> e.g., `edit module m/CS2103 mn/CS2105`
+**Edit Student** | `edit student m/<module name> i/<student id> [n/<edited student name>] [t/<edited student tele handle>] [e/<edited student email>]` <br> e.g., `edit student m/CS2100 i/A0123457A t/@amylee e/amy@u.nus.edu`
```

Comment:
maybe after the bug fix we wouldn't need m/<module name> anymore

Reply:
This is related to the change of removing module name `m/` parameter from edit student command yes? I'll open an issue for that to be fixed in a separate PR

Reply:
#216